spec/models/customer_spec.rb

```diff
@@ -0,0 +1,49 @@
+require 'rails_helper'
+RSpec.describe Customer, type: :model do
+  subject { Customer.new(first_name: "Jack", last_name: "Smith", phone: "8889995678", email: "jsmith@sample.com" )}
+  it "is valid with valid attributes" do
+    expect(subject).to be_valid
+  end
+  it "is not valid without a first_name" do
+    subject.first_name=nil
+    expect(subject).to_not be_valid
+  end
+  it "is not valid without a last_name" do
+    subject.last_name=nil
+    expect(subject).to_not be_valid
+  end
+  it "is not valid without a phone number" do
+    subject.phone=nil
+    expect(subject).to_not be_valid
+  end
+  it "is not valid without an email" do
+    subject.email=nil
+    expect(subject).to_not be_valid
+  end
+
+  it "is not valid if the phone number is not 10 chars" do
+    subject.phone = "12345"  # Less than 10 digits
+    expect(subject).to_not be_valid
+
+    subject.phone = "12345678901"  # More than 10 digits
+    expect(subject).to_not be_valid
+
+    subject.phone = "1234567890"  # Exactly 10 digits
+    expect(subject).to be_valid
+  end
```

Comment:
It's good to be thorough with your tests, but when there are multiple assertions in a row -- especially when they're all independent of each other -- we typically separate them into separate scenarios. (If you wish to group multiple scenarios together, you can do so with contexts as shown in [RSpec's examples](https://rspec.info/features/3-13/rspec-core/example-groups/basic-structure/))

The problem with multiple assertions in a row is that as soon as one fails, the scenario stops and you won't know what would've happened with the rest of the assertions. So in this case, if a bug caused `12345` to be valid, you wouldn't know whether `12345678901` or `1234567890` would pass or fail, and that is information that you'd want from the tests.

In fact, the helpful comments you added are a further indication that it's worth splitting into multiple scenarios.

```suggestion
  it "is not valid if the phone number has less than 10 digits" do
    subject.phone = "12345"
    expect(subject).to_not be_valid
  end
  
  it "is not valid if the phone number has more than 10 digits"
    subject.phone = "12345678901"
    expect(subject).to_not be_valid
  end
  
  it "is valid if the phone number has exactly 10 digits"
    subject.phone = "1234567890"
    expect(subject).to be_valid
  end
```